sound/soc/sof/intel/hda-stream.c

```diff
@@ -244,6 +257,17 @@ int hda_dsp_stream_trigger(struct snd_sof_dev *sdev,
 					SOF_HDA_SD_CTL_DMA_START |
 					SOF_HDA_CL_DMA_SD_INT_MASK, 0x0);
 
+		ret =
+		snd_sof_dsp_read_poll_timeout(sdev, HDA_DSP_HDA_BAR,
```

Comment:
it's better you have the function on the previous line. Try to see if you can fix the 80 char issue otherwise it's ok to not follow that rule.

Reply:
OK, thanks!